src/stampit.js

```diff
@@ -128,12 +131,9 @@ function extractStatics(targ, src) {
  */
 function compose(...factories) {
   let stamps = factories.slice(),
-      result = stampit(),
-      state = { state: {} },
-      methods = {},
-      statics = {};
-
-  result.compose = compose;
+      result = stampit.init(),
```

Comment:
Yeh, I am using this only to show off of one of the new v2 shortcuts. It could also be created using `stampit()` or `stampit().init`.